src/app/search/directives/search-series-form.component.ts

```diff
@@ -1,7 +1,6 @@
 import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
 import { Subject } from 'rxjs/Subject';
 import { SearchSeries } from '../search-series.model';
-import { CATEGORIES, SUBCATEGORIES } from '../../shared/model/itunes.categories';
```

Comment:
should we remove all mention of genre from the model too? 
<img width="393" alt="screen shot 2016-12-27 at 10 26 03 am" src="https://cloud.githubusercontent.com/assets/16007382/21502597/0238ceaa-cc1f-11e6-9632-5c4848875e14.png">